rapidsai/devel-centos.Dockerfile

```diff
@@ -80,6 +80,10 @@ RUN gpuci_mamba_retry create --no-default-packages --override-channels -n rapids
       -c nvidia \
       -c conda-forge \
       -c gpuci \
+      c-compiler \
+      cxx-compiler \
+      gcc_impl_linux-64=9 \
+      sysroot_linux-64=2.17 \
```

Comment:
Pinning to match the GCC & GLIBC versions set [in the `integration` repo]( https://github.com/rapidsai/integration/blob/b64460ece71b2c4b960e2d45c39b3d50049edfab/conda/recipes/versions.yaml#L22-L26 ) as part of PR ( https://github.com/rapidsai/integration/pull/484 )